.github/workflows/aro-hcp-dev-env-cd.yml

```diff
@@ -299,6 +299,11 @@
             cd maestro/
             make deploy-server
 
+        - name: 'Deploy Image Sync - OC Mirror'
```

Comment:
We'll only want to deploy this to a single service cluster, right?

Reply:
Yes. The chosen one for dev is the the MVP SC